02-microarray/clustering_microarray_01_heatmap.Rmd

```diff
@@ -1,7 +1,7 @@
 ---
 title: "Clustering Data - Microarray"
 author: "CCDL for ALSF"
-date: "`r format(Sys.time(), '%B %Y')`"
+date: "Oct 2020"
```

Comment:
Style question: Before we were using full Month names. I personally liked it a bit better.

Reply:
I agree with @jashapiro here, I personally like the full month names better (and believe I have _mostly_ been using them throughout or at least trying to).

Reply:
Full month names more accessible?